### PR TITLE
Better README.md

### PR DESCRIPTION
**Pull request by hekmati**
Aligned with the latest changes (go & node.js versions, ability to work locally etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,16 +5,31 @@ We hope the process is going well for you so far.
...
-3. Open `the_big_bang.ipynb` and follow the instructions there. There are also stub versions for Node (big_bang.js) and Go (big_bang.go), you can choose based on your preferred programming language.
+3. Open `the_big_bang.ipynb` and follow the instructions there
 
-4. If for some reason you are having hard time working with Binder, you can always download `the_big_bang.py` and work locally in your favorite IDE!
 
-5. Once you are done, download the notebook file as .ipynb, or take the relevant go/js file and send it to: `michael.h@sygnia.co`
+Once you are done:
+1. Download the notebook file as .ipynb, or take the relevant py/go/js file
+2. Zip the file
+3. Mail it to: `michael.h@sygnia.co`
```
Comment (ofir123) [inline README.md]: Can you ask Sapir to create an email group for this instead?
That way we can control it easily and add/remove people as we see fit (like our tower support group for example)

**Review by ofir123 — APPROVED** [review]
Great!

File: README.md (hunk 1 above)
Comment (hekmati) [reply README.md]: Great idea! Will do.

File: README.md [hunk 2]
```diff
@@ -5,16 +5,31 @@ We hope the process is going well for you so far.
...
+
 ## Instructions
 
+If you prefer to develop locally: 
+1. Clone the repo
+2. Choose the relevant file based on your preferred programming language
+3. Follow the instructions in the file
+
+
+If **Python** is your favorite language, and you prefer working with Jupyter Notebook, you can also use **Binder**:
```
Comment (yoavk) [inline README.md:22]: Do we want this to appear *before* the local option? personally I think it's more fun to do it in jupyter...

**Review by yoavk — APPROVED** [review]
👑

File: README.md (hunk 2 above)
Comment (hekmati) [reply README.md:22]: IMO it's better to go from general to specific (also - lately most of the candidates preferred to work locally).

**Pull request merged**